searx/settings.yml

```diff
@@ -152,6 +152,19 @@ outgoing:
 #   - "HTTPS rewrite"
 #   - ...
 
+# Example to rewrite result hostnames or remove results based on the hostname
+#
+# enabled_plugins:
```

Comment:
The `enabled_plugins` example is already in line 151 ... add `Hostname replace` to the examples from that line.

Do like to add the new plugin to ...  _default & values_

https://github.com/searxng/searxng/blob/76e0f6807ccddb12ca9efe3b2343735d6e1777d4/docs/dev/search_api.rst#L107-L113

and add plugin to _values_:

https://github.com/searxng/searxng/blob/76e0f6807ccddb12ca9efe3b2343735d6e1777d4/docs/dev/search_api.rst#L97-L105

BTW you could drop the surrounding `[...]` from the latter.

Reply:
done